meeting_minutes/221027_SARIF_TC_69.md

```diff
@@ -0,0 +1,138 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                    |
+|:-----------|:----------|:------------------|:---------------------------|
+| Charles    | Wilson    | Motional AD       | Member                     |
+| David      | Keaton    | Individual        | Chair                      |
+| Johnathan  | Gilday    | Contrast Security | Voting Member              |
+| Mary       | Martin    | Microsoft         | Voting Member              |
+| Michael    | Fanning   | Microsoft         | Voting member, taking notes|
+| Nathan     | Baird     | Microsoft         | Voting Member              |
```

Comment:
Did Nathan join after roll call?  Just checking because I don't have him on my list.

Reply:
I don't remember seeing him.

Reply:
Thanks for catching this.